harmony_service_lib/cli.py

```diff
@@ -130,7 +130,7 @@ def _write_error(metadata_dir, message, category='Unknown', level='Error'):
             json.dump(error_data, file)
 
 
-def _build_adapter(AdapterClass, message_string, sources_path, data_location, config):
+def _build_adapter(AdapterClass, message_string, sources_path, data_location, cfg):
```

Comment:
The name change is just to avoid conflicts with the name of the `harmony_service_lib.util.config` function.